cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -188,9 +192,12 @@ public static void editConfig() throws Exception {
             while (true) {
                 System.out.println(
                         "\n--------Chose the preferred IDE --------\n 1.for vscode \n 2.for eclipse \n 3.for IntelliJ_IDEA ");
-
-                id = Integer.parseInt(reader.readLine());
-
+                String ideInput = reader.readLine().trim();
+                if(ideInput==null || ideInput.isEmpty()){
+                    System.out.println("No input provided for IDE:Now using Existing/Default Config");
```

Comment:
this should never happen as the values of those equals what is in the config file.

Reply:
Sorry, I didn't get you. Just to clarify `ide` (which comes from config file )and `ideInput` are two different variables .

Reply:
okey then what it should say is "Empty/blank input provided - reseting to default setting" or similar.

Reply:
Sure, I'll modify  the console message .